yarn.lock

```diff
@@ -0,0 +1,5756 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I have removed the `yarn.lock` for the time being, as it's far from clear which lockfile (yarn/npm@5) we'll use and when it's appropriate to commit them.